My project/Assets/Scripts/Debug/ActionDebugger.cs

```diff
@@ -0,0 +1,74 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ActionDebugger : MonoBehaviour
+{
+    private CreatureData data;
+    [SerializeField]
+    private Rigidbody2D rb;
+    [SerializeField]
+    private RangeScanner scanner;
+    public ActionGraph graph { get; private set; }
+    public ActionNode current_action_node { get; private set; }
+    // Start is called before the first frame update
+    void Start()
+    {
+        //Creating Randomized Data
+        data = new(1001, 100, Random.Range(30, 40), 8, new Color(Random.Range(0f, 1f), Random.Range(0f, 1f), Random.Range(0f, 1f)), transform);
+
+        //Setting the action tree
+        CreateActions();
+        current_action_node = graph.root;
```

Comment:
typically within a project you stick to a single variable/class naming convention. for C# that's camel case. so this would be currentActionNode. it would be worth putting up a single PR to fix all of these within the project before we proceed